### DISCUSSION TITLE
Malformed stride.core.targets

### DISCUSSION THREAD

**Discussion by IXLLEGACYIXL** [stride3d/stride#1961] [Q&A]
ive made in a new folder

- git clone stride ( the one from today )
- msbuild /t:Restore Stride.sln
- changed in object id something
- dotnet build assemblyprocessor solution
- ran all tests in VS 

it results in 12 error of not finding a path for assemblyprocessor exe
![grafik](https://github.com/stride3d/stride/assets/107197024/2c53a21f-0977-455d-93f2-b89af099ca88)

```
  <Target Name="_StridePrepareAssetCompiler">
    <PropertyGroup>
      <!-- First try NuGet layout, then git checkout -->
      <StrideCompileAssetCommand Condition="'$(StrideCompileAssetCommand)' == '' And Exists('$(MSBuildThisFileDirectory)..\lib\net6.0-windows7.0\')">$(MSBuildThisFileDirectory)..\tools\net6.0-windows7.0\Stride.Core.Assets.CompilerApp.exe</StrideCompileAssetCommand>
      <StrideCompileAssetCommand Condition="'$(StrideCompileAssetCommand)' == '' And Exists('$(MSBuildThisFileDirectory)..\Stride.Core.Assets.CompilerApp.csproj')">$(MSBuildThisFileDirectory)..\bin\$(Configuration)\net6.0-windows7.0\Stride.Core.Assets.CompilerApp.exe</StrideCompileAssetCommand>
    </PropertyGroup>
```
this line fails

changing net6.0-windows to net6.0 lets it work again
assemblyprocessor doesnt even target net6 windows

did i make something wrong with my fresh clone? did i build assemblyprocessor wrong? does it happen for others? 

it seems when you had allready the repo cloned this issue doesnt appear, but a fresh clone lets it make appear
i can resovle it by changing the path but i dont get why it happened.. stride work the entire time

**Comment by ly3027929699** [comment]
It has no exe for me becasue the commits yesterday

**Comment by IXLLEGACYIXL** [comment]
> It has no exe for me becasue the commits yesterday

can you adjust the target file, open assembly processor sln and rebuild it and then try to run the tests?


in bith xml Elements you have to remove Windows

**Comment by Ethereal77** [comment]
Maybe that part (`net6.0-windows7.0`) shouldn't be hardcoded?
In the Stride solution we could use `$(StrideEditorTargetFramework)` so we can update it in just one place.
In user projects, the `Stride.Core.targets` should define the TFM (if it doesn't already, I can't remember), and use it in place of the hardcoded one.

Anyway, note I'm preparing a PR (#1904) right now that tries to integrate AssemblyProcessor in the main solution, so there will be no real need to compile it separately.

**Reply by IXLLEGACYIXL** [reply]
thank you, then i will keep the configuration locally and wait for your PR as i cant build stride else
